resource/environment/integration.js

```diff
@@ -159,20 +155,15 @@ const upgradeStack = Promise.coroutine(function* ( slack, envelope ) {
 	}
 
 	//get the template
-	const template = yield getTemplate( githubToken, githubOwner, githubRepo, branch, catalogNum);
+	const template = yield getTemplate( githubToken, githubOwner, githubRepo, branch, catalogNum).catch( () => undefined );
 	if ( template === undefined ) { 
-		return {
-			status: 404,
-			data: {
-				message: "Unable to get information from github"
-			}
-		};
+		return {status: 404, data: {message: "Unable to get information from github"}};
```

Comment:
the error message here should be more explicit for the user to understand what went wrong. What kind of information was unable to be found from github?